app.py

```diff
@@ -172,7 +180,11 @@ def delete_entry():
         Later I'll replace with a database algorith"""
 
         index = int(flask.request.form["Delete"])
-        print(index)
+        # Later, I'll store the following algorithm in another file
+        entry = Entry.query.filter_by(id=index)
```

Comment:
Here, we'll pull the entry object, then delete that object in the following lines.